flink-table/flink-table-api-java/src/test/java/org/apache/flink/table/test/program/TableTestProgram.java

```diff
@@ -95,12 +95,15 @@ public class TableTestProgram {
     /** Steps to be executed for running the actual test. */
     public final List<TestStep> runSteps;
 
+    private boolean isTerminating;
```

Comment:
let's drop that

Reply:
Removed and also removed the associated getter/setter